recipe/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pdfium-binaries" %}
+{% set version = "118.0.5868" %}
+{% set patch = version.split(".")[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bblanchon/{{ name }}/releases/download/chromium%2F{{ patch }}/{{ name.split('-')[0] }}-v8-mac-arm64.tgz  # [osx and arm64]
```

Comment:
when we upgrade the package, the tgz will have the same name, because it is not versioned (only the folder includes the patch number), and building it will return a hash mismatch error. we can solve it by using the `fn` field to rename the file in conda pkg cache, to something like `pdfium-binaries-{{ version }}-v8-mac.arm64.tgz`

Reply:
forgot to add the reference link: https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#filename